app/controllers/incidents_controller.rb

```diff
@@ -88,12 +102,6 @@ def mark_as
 
   def update
     set_incident
-    if @incident.update(incident_params)
-      flash[:success] = "Incident was successfully updated."
-      redirect_to @incident
-    else
-      render :edit
-    end
```

Comment:
Why are you removing this logic? This would make it so that incidents don't get persisted to the database :O